File: app_i/src/main/java/com/recoveryrecord/surveyandroid/example/ui/loadingpage/SplashScreenActivity.java
package com.recoveryrecord.surveyandroid.example.ui.loadingpage;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

import com.recoveryrecord.surveyandroid.example.ui.LoginActivity;

public class SplashScreenActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d ("Servicestatus", getPackageName());
        startActivity(new Intent(SplashScreenActivity.this, LoginActivity.class));
        finish();
    }
}

File: app_i/src/main/java/com/recoveryrecord/surveyandroid/example/SplashScreenActivity.kt
package com.recoveryrecord.surveyandroid.example;

import android.content.Context;
import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.drawable.BitmapDrawable;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.util.Log;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

public class SplashScreenActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d ("Servicestatus", getPackageName());
        startActivity(new Intent(SplashScreenActivity.this, LoginActivity.class));
        finish();
    }
}
